core/src/main/java/dev/streamx/cli/exception/UnableToConnectIngestionServiceException.java

```diff
@@ -9,7 +9,7 @@ public class UnableToConnectIngestionServiceException extends RuntimeException {
                 
         Verify:
          * if the mesh is up and running,
-         * if the ingestion service URL is set correctly \
+         * if the ingestion service URL is set correctly\
          (if it's not - set proper '--ingestion-url' option)""";
```

Comment:
This caused double space - now there is a single space (note that " (if" is already indented with one space)